Makefile

```diff
@@ -67,27 +67,27 @@ run-server-containerized:
 
 .PHONY: docker-compose
 docker-compose:
-	${COMPOSE_RUNTIME} -f tools/docker-compose/compose.yaml up --remove-orphans
+	${COMPOSE_RUNTIME} -f ${PWD}/tools/docker-compose/compose.yaml up --remove-orphans
 
 .PHONY: start-db
 # Run db in container for running Django application from source
 start-db:
-	${COMPOSE_RUNTIME} -f tools/docker-compose/compose-db.yaml up --remove-orphans -d
+	${COMPOSE_RUNTIME} -f ${PWD}/tools/docker-compose/compose-db.yaml up --remove-orphans -d
 
 .PHONY: stop-db
 # stop db container
 stop-db:
-	${COMPOSE_RUNTIME} -f tools/docker-compose/compose-db.yaml up down
+	${COMPOSE_RUNTIME} -f ${PWD}/tools/docker-compose/compose-db.yaml down
```

Comment:
This was not related to the `podman-compose` issue.  But it did not work unless I removed `up` :-)